frontend/common/src/components/form/Checkbox/Checkbox.tsx

```diff
@@ -79,7 +79,7 @@ export const Checkbox: React.FunctionComponent<CheckboxProps> = ({
             data-h2-align-items="b(center)"
             style={{ width: "100%" }}
           >
-            <label>
+            <label htmlFor={name}>
```

Comment:
Shouldn't this be `id`?

Reply:
I think either could work, let us ask Eric
@esizer 
Any thoughts on this, which it should be?

Reply:
Yeah, the `htmlFor` attribute should reference the `id` of the associated input and must be unique as well.